lib/logstash/plugin_mixins/enterprise_search/manticore_transport.rb

```diff
@@ -45,7 +45,7 @@ def manticore_transport_klass
 
     def transport_klass
       case Elasticsearch::Transport::VERSION
-      when /7\.1[123]/
+      when /^7\.1[123]/
```

Comment:
Example test failure:
```console
logstash-1           |   1) LogStash::PluginMixins::EnterpriseSearch::ManticoreTransport Client class #transport should use manticore setting the :ssl argument
logstash-1           |      Failure/Error: expect(result.transport).to be_a(Elastic::Transport::Transport::HTTP::Manticore)
logstash-1           |        expected #<Elasticsearch::Transport::Client:0x49afa057 @api_key=nil, @send_get_body_as="GET", @opaque_id_prefi...failure=>false, :randomize_hosts=>false, :http=>{:user=>nil, :password=>nil}, :retry_on_status=>[]}> to be a kind of Elastic::Transport::Transport::HTTP::Manticore
logstash-1           |      # ./spec/unit/outputs/manticore_transport_spec.rb:46:in `block in <main>'
```
I suppose this would actually be a bug in certain configurations of elastic stack and this plugin. I imagine this would need a release note?

Reply:
This is good find @donoghuc!
Ideally, using `major == 7 && minor > 1` to send the `Elasticsearch::Client` would be better but for the quick fix I am okay with this option.